### PR TITLE
[IP-284] enforce routemaster delivery timeouts

### PR DESCRIPTION
**Pull request by rxbynerd**
sets faraday options for a 5 second overall request timeout, and a 2 second connect timeout

_[Read more on JIRA!](https://deliveroo.atlassian.net/browse/IP-284)_

### CODE REVIEW

File: routemaster/services/deliver.rb [hunk 1]
```diff
@@ -40,6 +43,8 @@ def run
           response = _conn.post do |post|
             post.headers['Content-Type'] = 'application/json'
             post.body = data.to_json
+            post.options.timeout = TIMEOUT
+            post.options.open_timeout = CONNECT_TIMEOUT
```
Comment (mezis) [inline routemaster/services/deliver.rb]: Any reason to make these part of the query code, as opposed to configuring them when configuring the Faraday connection (which gets reused?)

Comment (rxbynerd) [reply routemaster/services/deliver.rb]: Hmm, I'll dig further into how Faraday & make sure these parameters are copied across from the request config - it was my understanding from the documentation this didn't happen, but I'll confirm.

Comment (rxbynerd) [reply routemaster/services/deliver.rb]: These are now set on the connection instead.

**Comment by mezis** [comment]
LGTM :+1:

**Pull request merged**